### ISSUE TITLE
How to download all NuGet packages at once?

### ISSUE THREAD

**Issue by lg2de** [dotnet/extensions#3111]
### Is your feature request related to a problem? Please describe.
As a software vendor we have to ensure that all packages included in our software have a license according to our rules.
Therefore we started approval process for NuGet packages.

So far this is working fine for individual packages based on .NET Framework (4.X).
Starting with .NET Core we are facing a huge list of packages required to be approved.

### Describe the solution you'd like
I would like to have a download package containing all NuGet packages of a feature set, e.g. all from `Microsoft.Extensions` in version 3.1.0.
Within the (ZIP) package all packages should be grouped by the license.
If all have the same license: Fine, no grouping required.

### Describe alternatives you've considered
Currently every package needs to be downloaded manually from nuget.org or copied from local nuget cache.
Further every package needs to be check for the license.

### Additional context
In this context the following question may be relevant:
Why all packages are released with new version even the package has not changed since last version?
Especially the abstractions packages (e.g. microsoft.extensions.logging.abstractions) are released as 3.1.0, 3.1.1, 3.1.2 and 3.1.3 even there **must** be no change on the abstractions (according SemVer), isn't it?

**Comment by Pilchie** [comment]
@leecow this sounds like a general release feature request for all of .NET Core - is there a better repo for it?

**Comment by leecow** [comment]
Possibly core-setup since it's a package building exercise. @NikolaMilosavljevic and @dleeapho, does that sound right?

**Comment by dleeapho** [comment]
Yes, dotnet/runtime (area-setup) sounds good but dotnet/installer could make a credible claim too, thoughts @KathleenDollard?

**Comment by anurse** [comment]
I'm going to move this to dotnet/runtime area-setup for now as suggested above.
